### ISSUE TITLE
Feature request: Global FPS variable

### ISSUE THREAD

**Issue by marenz2569** [vifino/sled#16]
I think it is useful if a global FPS variable would be introduced and all animations would use it, well except the ones which do not update the screen for a certain amount of time, like gfx_error.

**Comment by vifino** [comment]
No, this is actually a feature, as we explicitly allow variable refresh rate.

Effects that have no new content available shouldn't have to redraw with no change.
ws2812b's really benefit from the variable refresh rate, as we send updates on demand, no vsync or other timing limits there.

So, well, I don't agree with "all animations".
Perhaps some that can render at really high speeds, those locked at 60 right now, for example.

**Comment by marenz2569** [comment]
Yes this is what I mean. A maximum FPS should be given by the output module.
All other animations that don't need to redraw shall just use there set FPS value.
